core/templates/pages/topic-editor-page/subtopic-editor/subtopic-editor-tab.component.ts

```diff
@@ -278,15 +281,22 @@ angular.module('oppia').component('subtopicEditorTab', {
         ctrl.schemaEditorIsShown = false;
         $scope.$on(EVENT_TOPIC_INITIALIZED, _initEditor);
         $scope.$on(EVENT_TOPIC_REINITIALIZED, _initEditor);
-        $scope.$on(EVENT_SUBTOPIC_PAGE_LOADED, function() {
```

Comment:
Why is this not needed? I mean I could see the $broadcast corresponding to it.

Reply:
Not needed as in? I've replaced it by a subscribe